docs/development/tutorial/README.md

```diff
@@ -0,0 +1,56 @@
+# NEW DEVELOPER GETTING STARTED!
+
+## Assumptions
+
+- Some familiarity with 'git' is assumed
+- Some familiarity with a favorite
+
+## Setup Instructions
+
+- Fork the triplea repository: <https://github.com/triplea-game/triplea>
+- Using a Git client (command line or GUI like 'github desktop'), clone
+  your fork locally, eg:
+  ```git clone git@github.com:<your username>/triplea.git```
+  
+- Setup IDE: [/docs/development/how-to/ide-setup](../how-to/ide-setup>)
+
+## Further, typical git commands and submitting PRs
+
+- [/docs/development/how-to/typical-git-workflow](../how-to/typical-git-workflow.md)
+- [/docs/development/reference/dev-process/pull-requests.md](../reference/dev-process/pull-requests.md)
+
+
+
+## Running the code
+
+- IDE (there are launchers checked in for convenience)
+- CLI: `./gradlew :game-app:game-headed:run`
```

Comment:
I guess this is Linux (and potentially MacOS) only.
The sources contains a `gradlew.bat` which I guess is the Windows equivalent?

Reply:
`gradlew.bat` is indeed the windows equivalent
I updated assumptions to note that we presume someone is going to go ahead and run 'cygwin'. If the windows CLI command is really straight forward and works just fine, it'd be fine to add to the documents. 

Notably it's the responsibility of anyone to ensure any setup and configuration that they use is documented. This avoids others from having to again re-invent what is already known by someone else. You might tell that I use linux (and happily Mac OS is basically the same from a CLI point of view).